packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -247,6 +247,17 @@ export class CdkErrorMapper {
       errorName: 'SecretNotSetError',
       classification: 'ERROR',
     },
+    {
+      // This error originates from AppSync when there are syntax/type errors or unsupported features being used in resolvers
+      errorRegex:
+        /The code contains one or more errors. \(Service: AppSync|BadRequestException: The code contains one or more errors./,
+      humanReadableErrorMessage:
+        'There are one or more errors with data handlers.',
+      resolutionMessage:
+        'Check your handlers defined in your data backend for syntax errors, type errors, and unsupported features of AppSync resolvers.',
```

Comment:
How will the customers find the actual error? That is, how would they troubleshoot?

Reply:
The original error message is extremely vague and the stack trace is not helpful (points to `aws-cdk` in node_modules and not anywhere in amplify project).

Reply:
@aws-amplify/amplify-data, how can customers debug when this error happens?